isis/src/control/objs/CsmBundleObservation/CsmBundleObservation.cpp

```diff
@@ -873,4 +873,24 @@ QString CsmBundleObservation::formatBundleOutputString(bool errorPropagation, bo
 
     return "";
   }
+
```

Comment:
whoops, I just thought I'd left this file out to flesh it out later, but this is a (very) old version. I'll update it.